cheatsheets_draft/Secrets_Management_CheatSheet.md

```diff
@@ -154,16 +155,47 @@ Given that the CI/CD tooling heavily consume secrets, it is key that the pipelin
 - Have proper authentication, authorization and accounting in place.
 - Make sure that pipelines can only be created by means of an approved process, including MR/PR steps to make sure that a created pipeline is security-reviewed.
 
-### 3.2. Where should a secret be?
+
+### 3.2 Where should a secret be?
+
+- DO YOU PUT THE SECRET IN GIT PROVISIONGIN SYSTEM?J
+- DO YOU PUT THE SECRET IN YOUR BUILDTOOL?
+- DO THESE SYSTEMS HAVE A LINK TO A SECRETS MANAGEMENT SYSTEM WHICH HAS THE SECRET?
 
 A pipeline should not store production secrets. Instead, it should only host temporal credentials (E.g. often rotated credentials) to the secret management-system. TODO: CONTINUE HERE!  
 
 
-### 3.3. Rotation vs Dynamic Creation
+#### 3.2.1 Using Git provisioning system
+
+EXPLAIN DO'S AND DON'TS
+
+#### 3.2.2 Using the pipeline tool
+
+EXPLAIN DO'S AND DON'TS
+
+#### 3.2.3 Storing it in a secrets management system
+
+EXPLAIN DO'S AND DON'TS
+
+### 3.3: Authentication and Authorization
+
+HOW DOES A PIPELIEN AUTHENTICATE? HOW DO YOU KNOW AUTHORIZAITON IS OK?
```

Comment:
```suggestion
HOW DOES A PIPELINE AUTHENTICATE? HOW DO YOU KNOW AUTHORIZATION IS OK?
```

Reply:
removed the text as this was a placeholder for the actual text.